### PR TITLE
Track analytics for XMTP Labs employee usage on alpha.xmtp.chat only

### PR DESCRIPTION
**Pull request by bhavya2611**
This PR adds Google Analytics tracking to xmtp-inbox-web code deployed to alpha.xmtp.chat only. This is for temporary testing purposes and only tracks analytics for explicitly listed XMTP Labs employee wallets.

xmtp-inbox-web does NOT use Google Analytics on any other domains, including xmtp.chat.

### CODE REVIEW

File: hooks/useSendMessage.ts [hunk 1]
```diff
@@ -40,6 +42,15 @@ const useSendMessage = (conversationId: string) => {
         }
       }
       await selectedConversation?.send(message);
+      /* The function below, emitMsgSentEvent will only be called for specific
```
Comment (daria-github) [inline hooks/useSendMessage.ts:45]: super nit - typo

**Review by daria-github — APPROVED** [review]

**Comment by jhaaaa** [comment]
Thank you for this @bhavya2611. Just to make the use case super clear, I'd change the PR title and description as follows:

**Title**: Track analytics for XMTP Labs employee usage on alpha.xmtp.chat only

**Description**: This PR adds Google Analytics tracking to xmtp-inbox-web code deployed to alpha.xmtp.chat only. This is for temporary testing purposes and only tracks analytics for explicitly listed XMTP Labs employee wallets.

xmtp-inbox-web does NOT use Google Analytics on any other domains, including xmtp.chat.

File: helpers/internalTracking.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { fetchEnsName } from "@wagmi/core";
+import { address } from "../pages/inbox";
+import { isAppEnvAlpha } from "./env";
+
+/* The functions below will only be called for specific 
+wallets of XMTP-LABS team members using the interal domain alpha.xmtp.chat 
+to gather insights about user behaviour which can help the team 
+to build a better app. */
+
+// List of XMTP LABS Team Wallets
```
Comment (jhaaaa) [inline helpers/internalTracking.ts:11]: Just some suggested slight tweaks @bhavya2611:

/* The functions below are called only when 
specific XMTP Labs team wallets use 
the internal domain alpha.xmtp.chat. This 
tracking is temporary and meant to help
surface insights about team usage to 
help build a better app. */

// List of XMTP Labs team wallets

File: pages/index.tsx [hunk 3]
```diff
@@ -26,9 +28,17 @@ const OnboardingPage: NextPage = () => {
...
   useEffect(() => {
-    if (address && !newAccount && client) {
-      router.push("/inbox");
-    }
+    const routeToInBox = async () => {
+      if (address && !newAccount && client) {
+        /* The function below, emitPageVisitEvent will only be called for specific 
+          wallets of XMTP-LABS team members using the interal domain alpha.xmtp.chat 
+          to gather insights about user behaviour which can help the team 
+          to build a better app. */
```
Comment (jhaaaa) [inline pages/index.tsx:36]: Just to add to Daria's comment, some suggested tweaks:

/* The emitPageVisitEvent function is called only when 
specific XMTP Labs team wallets use 
the internal domain alpha.xmtp.chat. This 
tracking is temporary and meant to help
surface insights about team usage to 
help build a better app. */

**Review by daria-github — APPROVED** [review]

**Pull request merged**